drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/MetaDataColumnPage.java

```diff
@@ -19,28 +19,29 @@
 import javax.enterprise.context.Dependent;
 import javax.inject.Inject;
 
-import com.google.gwt.user.client.ui.Widget;
+import com.google.gwt.regexp.shared.RegExp;
 import org.drools.workbench.screens.guided.dtable.client.resources.i18n.GuidedDecisionTableErraiConstants;
 import org.drools.workbench.screens.guided.dtable.client.wizard.column.pages.common.BaseDecisionTableColumnPage;
 import org.drools.workbench.screens.guided.dtable.client.wizard.column.plugins.MetaDataColumnPlugin;
 import org.jboss.errai.ui.client.local.spi.TranslationService;
 import org.uberfire.client.callbacks.Callback;
 import org.uberfire.client.mvp.UberElement;
-import org.uberfire.client.mvp.UberView;
 
 import static org.drools.workbench.screens.guided.dtable.client.wizard.column.pages.common.DecisionTableColumnViewUtils.nil;
 
 @Dependent
 public class MetaDataColumnPage extends BaseDecisionTableColumnPage<MetaDataColumnPlugin> {
 
     private View view;
+    private RegExp containWhiteSpacePattern;
 
     @Inject
     public MetaDataColumnPage(final View view,
                               final TranslationService translationService) {
         super(translationService);
 
         this.view = view;
+        this.containWhiteSpacePattern = RegExp.compile("\\s");
```

Comment:
@jomarko TBH I didn't know there was _any_ restriction: https://docs.jboss.org/drools/release/6.5.0.Final/drools-docs/html/ch08.html#d0e8397 makes no mention of reserved characters. @mariofusco can you advise the restrictions on rule meta-data?